Dockerfile.public

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.19.0-alpine as build
+WORKDIR /app
+COPY . .
+RUN env CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o /rds_exporter ./
+RUN chmod +x /rds_exporter
+
+FROM alpine:latest
```

Comment:
so we already have: https://github.com/percona/rds_exporter/blob/main/Dockerfile.releaser

why this one is needed?

Reply:
This is use for my tests only so will be exclude too. Thanks.